### DISCUSSION TITLE
Doccano pip installation issue

### DISCUSSION THREAD

**Discussion by carla-fiadeiro** [doccano/doccano#1126] [Q&A] [answered]
Hi,

I tried to install doccano using the pip installation option, I followed the steps defined in:

```
pip installation
To install doccano, simply run:

pip install doccano
After installation, simply run the following command:

doccano
Go to http://0.0.0.0:8000/.
```
I am able to access Doccano but I'm asked for a user and a password and I haven't defined any during the installation process. What I am doing wrong?

Thank you,
Carla

**Comment by carla-fiadeiro** [comment]
HI, any ideas on my issue?

Thank you,
Carla

**Comment by ricjhill** [comment, marked as answer]
Hi  Carla.  There are admin details hardcoded in the repo for  docker. 

I  haven't tried pip , I'm on windows and need to change some paths before getting it to work

> 
> ADMIN_USERNAME: "admin"
> ADMIN_PASSWORD: "password"

**Reply by carla-fiadeiro** [reply]
It worked,
Thank you a lot :)

**Reply by icoxfog417** [reply]
@ricjhill Thank you for your contribution!

**Reply by greathour** [reply]
you can change password in  https://ip:8000/admin/,add user also,
after pip install ,can find in dictory python/lib/python3.x/site-package/app/
then python manage.py ...
